### PR TITLE
Updated changelog for 2.0.0-beta.1.

### LINKED ISSUE #348: Update CHANGELOG.md for 2.0
- Closed issues: https://github.com/GlotPress/GlotPress-WP/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aclosed+milestone%3A1.1+
- Closed PRs: https://github.com/GlotPress/GlotPress-WP/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+milestone%3A1.1

### PR DESCRIPTION
**Pull request by toolstack**
Resolves #348.

### CODE REVIEW

**Comment by toolstack** [comment]
I've left the PR's that haven't been merged at the bottom in case they get bumped, once they're merged we can add them back in to their proper spot.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,54 @@
+## 2.0.0-beta.1 (March 17, 2016)
+
+* BREAKING: In `gp_update_meta()`/`gp_delete_meta()` remove meta handling for users and options ([#300](https://github.com/GlotPress/GlotPress-WP/issues/300))
+* BREAKING: Remove deprecated `assets/img/glotpress-logo.png` ([#327](https://github.com/GlotPress/GlotPress-WP/issues/327))
+* BREAKING: Replace gp_sanitize_for_url() with wp's sanitize_title(). ([#330](https://github.com/GlotPress/GlotPress-WP/pull/330))
```
Comment (ocean90) [inline CHANGELOG.md]: All function/methods/class names should be in backticks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,54 @@
...
+* Bugfix: Merge similar translation strings and avoid using HTML tags in translation strings ([#295](https://github.com/GlotPress/GlotPress-WP/pull/295))
+* Bugfix: Actions for translations have no `gp_` prefix ([#232](https://github.com/GlotPress/GlotPress-WP/issues/232))
+* Bugfix: `$original->validate()` fails if singular is '0' ([#301](https://github.com/GlotPress/GlotPress-WP/issues/301))
+* Bugfix: Auto generation of a project slug doesn't work correctly in some cases ([#328](https://github.com/GlotPress/GlotPress-WP/issues/328))
+* Bugfix: Suspend cache invalidation during original imports ([#332](https://github.com/GlotPress/GlotPress-WP/issues/332))
+* Bugfix: Do not allow translations with empty plurals ([#308](https://github.com/GlotPress/GlotPress-WP/pull/308))
+* Bugfix: Update schema definitions to work with WP's dbDelta() ([#343](https://github.com/GlotPress/GlotPress-WP/issues/343))
+* Bugfix: Translation Set: Use the correct select helper for the edit form ([#351](https://github.com/GlotPress/GlotPress-WP/pull/351))
+* Delete: Remove composer from unit tests ([#251](https://github.com/GlotPress/GlotPress-WP/pull/251))
```
Comment (ocean90) [inline CHANGELOG.md]: This shouldn't be part of the changelog. It's not relevant for the end-user.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,54 @@
...
+
+* BREAKING: Move Translation Propagation into a plugin ([#337](https://github.com/GlotPress/GlotPress-WP/issues/337))
+* BREAKING: Don't store counts as a public property of GP_Translation_Set ([#334](https://github.com/GlotPress/GlotPress-WP/issues/334))
```
Comment (ocean90) [inline CHANGELOG.md:42]: 44 + 45 should be on to top too.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,54 @@
...
+* New: Make translation sets deletable ([#267](https://github.com/GlotPress/GlotPress-WP/issues/267))
```
Comment (ocean90) [inline CHANGELOG.md]: Should be on top too.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,54 @@
...
+* New: Display warning if unsupported version of PHP is detected ([#276](https://github.com/GlotPress/GlotPress-WP/issues/276))
+* New: Add UI to manage GP Administrators ([#233](https://github.com/GlotPress/GlotPress-WP/issues/233))
+* New: Add doc blocks for actions and filters ([#50](https://github.com/GlotPress/GlotPress-WP/issues/50))
+* New: Add auto detection of uploaded import files ([#290](https://github.com/GlotPress/GlotPress-WP/issues/290))
+* New: Add backend support to allow for integration with WP's user profile ([#196](https://github.com/GlotPress/GlotPress-WP/issues/196))
+* New: Things: Introduce `after_delete()` method and new actions ([#294](https://github.com/GlotPress/GlotPress-WP/issues/294))
+* New: Translations: Include check button for case-sensitive search ([#312](https://github.com/GlotPress/GlotPress-WP/issues/312))
+* New: Java properties file support ([#297](https://github.com/GlotPress/GlotPress-WP/issues/297))
+* New: Standardize triggers in projects, translations and originals ([#294](https://github.com/GlotPress/GlotPress-WP/issues/294))
+* New: Use wp-dev-lib for development ([#320](https://github.com/GlotPress/GlotPress-WP/issues/320))
```
Comment (ocean90) [inline CHANGELOG.md]: This shouldn't be part of the changelog. It's not relevant for the end-user.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,54 @@
+## 2.0.0-beta.1 (March 17, 2016)
+
+* BREAKING: In `gp_update_meta()`/`gp_delete_meta()` remove meta handling for users and options ([#300](https://github.com/GlotPress/GlotPress-WP/issues/300))
+* BREAKING: Remove deprecated `assets/img/glotpress-logo.png` ([#327](https://github.com/GlotPress/GlotPress-WP/issues/327))
+* BREAKING: Replace gp_sanitize_for_url() with wp's sanitize_title(). ([#330](https://github.com/GlotPress/GlotPress-WP/pull/330))
+* BREAKING: Enhance gp_meta_update return values ([#318](https://github.com/GlotPress/GlotPress-WP/issues/318))
+* BREAKING: Remove GP_CLI_WPorg2Slug ([#347](https://github.com/GlotPress/GlotPress-WP/issues/347))
+* New: File extension list for portable object message catalog should include .pot ([#230](https://github.com/GlotPress/GlotPress-WP/issues/230))
+* New: Update locale definitions ([#246](https://github.com/GlotPress/GlotPress-WP/issues/246))
+* New: Add CONTRIBUTING.md ([#216](https://github.com/GlotPress/GlotPress-WP/issues/216))
```
Comment (ocean90) [inline CHANGELOG.md]: This shouldn't be part of the changelog. It's not relevant for the end-user.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,54 @@
...
+* Bugfix: [_]wp_specialchars() is deprecated/private and should be replaced ([#280](https://github.com/GlotPress/GlotPress-WP/issues/280))
+* Bugfix: Merge similar translation strings and avoid using HTML tags in translation strings ([#295](https://github.com/GlotPress/GlotPress-WP/pull/295))
+* Bugfix: Actions for translations have no `gp_` prefix ([#232](https://github.com/GlotPress/GlotPress-WP/issues/232))
+* Bugfix: `$original->validate()` fails if singular is '0' ([#301](https://github.com/GlotPress/GlotPress-WP/issues/301))
+* Bugfix: Auto generation of a project slug doesn't work correctly in some cases ([#328](https://github.com/GlotPress/GlotPress-WP/issues/328))
+* Bugfix: Suspend cache invalidation during original imports ([#332](https://github.com/GlotPress/GlotPress-WP/issues/332))
+* Bugfix: Do not allow translations with empty plurals ([#308](https://github.com/GlotPress/GlotPress-WP/pull/308))
+* Bugfix: Update schema definitions to work with WP's dbDelta() ([#343](https://github.com/GlotPress/GlotPress-WP/issues/343))
+* Bugfix: Translation Set: Use the correct select helper for the edit form ([#351](https://github.com/GlotPress/GlotPress-WP/pull/351))
```
Comment (ocean90) [inline CHANGELOG.md:39]: That was a bug in 6a8a67104825e78bcfe7328b50e2037d0de01f0f, doesn't need to be listed here.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,54 @@
...
+* New: Standardize triggers in projects, translations and originals ([#294](https://github.com/GlotPress/GlotPress-WP/issues/294))
+* New: Use wp-dev-lib for development ([#320](https://github.com/GlotPress/GlotPress-WP/issues/320))
+* New: Public and private profiles do not have the same design ([#325](https://github.com/GlotPress/GlotPress-WP/issues/325))
+* New: Translation Sets: Validate slugs on saving ([#329](https://github.com/GlotPress/GlotPress-WP/issues/329))
+* Bugfix: Mark Sindhi locale as RTL ([#243](https://github.com/GlotPress/GlotPress-WP/issues/243))
+* Bugfix: Replace current_user_can( 'manage_options' ) with GP permissions ([#254](https://github.com/GlotPress/GlotPress-WP/issues/254))
+* Bugfix: Child projects are inaccessible if permalinks have trailing slash ([#265](https://github.com/GlotPress/GlotPress-WP/issues/265))
+* Bugfix: Various code cleanups to improve code quality ([#237](https://github.com/GlotPress/GlotPress-WP/issues/237))
+* Bugfix: Update readme files ([#228](https://github.com/GlotPress/GlotPress-WP/issues/228))
+* Bugfix: Tests: Remove unnecessary filters for `gp_set_cookie` ([#287](https://github.com/GlotPress/GlotPress-WP/issues/287))
```
Comment (ocean90) [inline CHANGELOG.md]: This shouldn't be part of the changelog. It's not relevant for the end-user.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,54 @@
...
+
+* BREAKING: In `gp_update_meta()`/`gp_delete_meta()` remove meta handling for users and options ([#300](https://github.com/GlotPress/GlotPress-WP/issues/300))
+* BREAKING: Remove deprecated `assets/img/glotpress-logo.png` ([#327](https://github.com/GlotPress/GlotPress-WP/issues/327))
+* BREAKING: Replace gp_sanitize_for_url() with wp's sanitize_title(). ([#330](https://github.com/GlotPress/GlotPress-WP/pull/330))
+* BREAKING: Enhance gp_meta_update return values ([#318](https://github.com/GlotPress/GlotPress-WP/issues/318))
+* BREAKING: Remove GP_CLI_WPorg2Slug ([#347](https://github.com/GlotPress/GlotPress-WP/issues/347))
+* New: File extension list for portable object message catalog should include .pot ([#230](https://github.com/GlotPress/GlotPress-WP/issues/230))
+* New: Update locale definitions ([#246](https://github.com/GlotPress/GlotPress-WP/issues/246))
+* New: Add CONTRIBUTING.md ([#216](https://github.com/GlotPress/GlotPress-WP/issues/216))
+* New: Support GitHub directory ([#283](https://github.com/GlotPress/GlotPress-WP/issues/283))
```
Comment (ocean90) [inline CHANGELOG.md]: This shouldn't be part of the changelog. It's not relevant for the end-user.

File: CHANGELOG.md (hunk 3 above)
Comment (toolstack) [inline CHANGELOG.md:42]: See my [note](https://github.com/GlotPress/GlotPress-WP/pull/354#issuecomment-197972434) about the lines at the bottom, I'll move them as we merge the PR's.

File: CHANGELOG.md (hunk 4 above)
Comment (toolstack) [inline CHANGELOG.md]: See my [note](https://github.com/GlotPress/GlotPress-WP/pull/354#issuecomment-197972434) about the lines at the bottom, I'll move them as we merge the PR's.

File: CHANGELOG.md (hunk 1 above)
Comment (toolstack) [inline CHANGELOG.md]: Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (toolstack) [inline CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 5 above)
Comment (toolstack) [inline CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 6 above)
Comment (toolstack) [inline CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 7 above)
Comment (toolstack) [inline CHANGELOG.md:39]: Removed.

File: CHANGELOG.md (hunk 8 above)
Comment (toolstack) [inline CHANGELOG.md]: Removed.

File: CHANGELOG.md (hunk 9 above)
Comment (toolstack) [inline CHANGELOG.md]: Removed.

**Pull request merged**